From: Jiri Slaby <jslaby@suse.cz>
Subject: XEN: add fixup_4gb_segment ENDPROC
Patch-mainline: never

There is missing ENDPROC for fixup_4gb_segment, add it now.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 arch/x86/kernel/entry_32-xen.S |    1 +
 1 file changed, 1 insertion(+)

--- a/arch/x86/kernel/entry_32-xen.S
+++ b/arch/x86/kernel/entry_32-xen.S
@@ -1235,6 +1235,7 @@ ENTRY(fixup_4gb_segment)
 	CFI_ADJUST_CFA_OFFSET 4
 	jmp error_code
 	CFI_ENDPROC
+ENDPROC(fixup_4gb_segment)
 
 .section .rodata,"a"
 #include "syscall_table.S"
